### DISCUSSION TITLE
How to use/upload your own "Severity assumptions"

### DISCUSSION THREAD

**Discussion by eabase** [neherlab/covid19_scenarios#427] [General]
In fact I don't even know if the "Severity assumptions"  are used in the model.

So I need to know:
1. Is it used, where and how?
2. If it is used, how can I upload/provide my own without having to edit the entire table every time I want to run this?
3. We know that the Chinese data is quite unreliabale, and I want to use the Italian data. How to do that?

Thanks.

**Comment by nnoll** [comment]
For now there isn't an easy way to bulk import assumptions into the severity table. As of today, there are two main ways that you can accomplish the input of your own assumptions:

(i) Enter them manually by hand. I don't believe these values are serialized with the URL as of now so there's no way to maintain this state which is unfortunate.
(ii) Fork the repo and replace src/assets/severity.json which contains all of our preset "assumptions" of the data. 

These options are, of course, bulky and suboptimal. We have a long standing issue of importing scenarios that hasn't been completed, however we are almost ready to tackle this when we complete input validation (which is our major hurdle right now).
